source/_posts/2018-07-11-fast-sudoku-solver-in-haskell-2-a-200x-faster-sol.markdown

```diff
@@ -5,6 +5,7 @@ created_at: 2018-07-11 00:00:00 UTC
 author: Abhinav Sarkar
 post_url: "https://abhinavsarkar.net/posts/fast-sudoku-solver-in-haskell-2/"
 layout: post
+hidden: true
```

Comment:
3 haskell blogs were showing up on the grid. Hides all but one for variety. They'll still show up on the archive/elsewhere.